src/Psalm/Internal/Analyzer/Statements/Expression/Call/NewAnalyzer.php

```diff
@@ -89,7 +90,7 @@ public static function analyze(
                 ) {
                     $codebase->file_reference_provider->addMethodReferenceToClassMember(
                         $context->calling_method_id,
-                        'use:' . $stmt->class->parts[0] . ':' . md5($statements_analyzer->getFilePath()),
+                        'use:'.$stmt->class->parts[0].':'.md5($statements_analyzer->getFilePath()),
```

Comment:
I find it more readable with spaces actually

Reply:
I agree but that leads to CS error. Could you advise what's the proper way of working this around?

Reply:
I don't see the CS warning locally. Please push the change, and we'll figure this out.

Reply:
Done